src/ChunkSender.h

```diff
@@ -53,87 +56,73 @@ class cChunkSender:
 {
 	typedef cIsThread super;
 public:
-	cChunkSender(void);
+	cChunkSender(cWorld & a_World);
 	~cChunkSender();
 
 	enum eChunkPriority
 	{
-		E_CHUNK_PRIORITY_HIGH   = 0,
-		E_CHUNK_PRIORITY_MEDIUM = 1,
-		E_CHUNK_PRIORITY_LOW    = 2,
+		E_CHUNK_PRIORITY_LEGENDARY = 0,
```

Comment:
I think it seems to fit in well enough, since you are going to have to go to the enum to work out relative priority anyway most of the time. What would be better would be if we could associate the priority of the original request, but I didn't want to have to work out the race conditions of that.